tests/integration_tests/test_source_faker_integration.py

```diff
@@ -315,3 +315,36 @@ def test_example_config_file(source_faker_seed_a: ab.Source) -> None:
             output_file=temp.name,
         )
         assert Path(temp.name).exists()
+
+
+def test_merge_insert_not_supported_for_duckdb(
```

Comment:
for some reason, i was not able to use a CacheBase fixture comprising all cache types, so wrote separate tests.

Reply:
Okay. I could probably help integrate with the generic tests, but what you have here seems sufficient so I don't think we need to prioritize it.